Makefile

```diff
@@ -54,7 +54,7 @@ lint:
 create_environment:
 	@pip install -q virtualenv virtualenvwrapper
 	@echo ">>> Installing virtualenvwrapper if not already intalled.\nMake sure the following lines are in shell startup file\n\
-	export WORKON_HOME=$$HOME/.virtualenvs\nexport PROJECT_HOME=$$HOME/Devel\nsource /usr/local/bin/virtualenvwrapper.sh\n"
+	export WORKON_HOME=$$HOME/.virtualenvs\nexport PROJECT_HOME=$$HOME/Devel\nsource $(shell which virtualenvwrapper.sh)\n"
```

Comment:
@ilazakis hey man, you needed to add `shell` as well. Otherwise, the message wouldn't be displayed.

Reply:
Ah I didn't check the console output, thanks @pm3310 😃